sokoban/src/components/LevelBoxes.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { LevelBox } from './LevelBox';
+import { IBoxes } from '../interfaces';
+import "./LevelBox.css"
+
+const boxes: IBoxes[] = [
+  { id: 0, levelVal: 1, stars: 'En av fem' },
```

Comment:
Ladda in globals och kör en loop så slipper du statisk inmatning här vid nya levels framöver.